plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeScanSession.kt

```diff
@@ -286,7 +286,13 @@ class CodeWhispererCodeScanSession(val sessionContext: CodeScanSessionContext) {
         )
     } catch (e: Exception) {
         LOG.debug { "Create Upload URL failed: ${e.message}" }
-        throw codeScanServerException("CreateUploadUrlException: " + e.message?.let { it } ?: message("codewhisperer.codescan.run_scan_error_telemetry"))
+
+        val errorMessage = when {
+            e.message?.contains("Your account is not authorized to make this call.") == true -> "Your account is not authorized to make this call."
+            e.message?.contains("Service returned HTTP status code 407") == true -> "Service returned HTTP status code 407"
+            else -> e.message ?: message("codewhisperer.codescan.run_scan_error_telemetry")
```

Comment:
are there any other exception cases we are missing here.

Reply:
I found these from the Kibana. As we get more metrics we need to add if required.